modules/optimization/src/functions/VectorNearestPointFunction.C

```diff
@@ -0,0 +1,197 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "VectorNearestPointFunction.h"
+
+registerMooseObject("OptimizationApp", VectorNearestPointFunction);
+
+InputParameters
+VectorNearestPointFunction::validParams()
```

Comment:
NearestPointsFunction ?

NearestReporterCoordinatesFunction ?

Reply:
done.  Zach deleted this object.

Reply:
done renamed NearestReporterCoordinatesFunction